modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -813,6 +902,34 @@ export default {
       } else {
         return true;
       }
+    },
+    trayItemTooltip(item) {
+      if (item.options.toggle) {
```

Comment:
I'm not sure why this is dependent on toggling. You could have a context menu in this location and still want the tooltip.

Reply:
It's not. See the last "else" clause.

Reply:
Got it. The layers had me confused.